fastlane/docs/Codesigning/XcodeProject.md

```diff
@@ -31,6 +31,18 @@ PROVISIONING_PROFILE_SPECIFIER = "[team_id]/[profile_name]";
 
 You could pass the profile specifier via environment variables too (See Xcode 7 approach below), however in most cases it's enough to define it statically in your project, per target, since usually neither your Team ID, nor the provisioning profile change.
 
+If you want to set it dynamically, you can use sigh or match to fill the environment variables:
```

Comment:
@KrauseFx Match pre-fills the environment but the env variable name is for every build type different. One think we could do is to remove the build type from the env variable. I think `sigh_tools.fastlane.test_profile_name` would be okey. The Bundle identifier is necessary to distinguish between targets (app, extensions,...) I think the build type is only necessary if you download the app store profile and the adhoc profile in the same lane. BTW Sigh is using only one variable for every profile (SIGH_UDID, SIGH_PROFILE_NAME).